modules/ip/datapath/ip_input.c

```diff
@@ -18,6 +18,17 @@
 #include <rte_mbuf.h>
 #include <rte_mbuf_dyn.h>
 
+#ifdef __GROUT_UNIT_TEST__
+// The function is defined as static inline in the original code, which means they are
+// expanded at compile-time and do not have linkable symbols. This makes them impossible
+// to wrap directly using CMocka's function wrapping mechanism.
+//
+// To work around this limitation, `#define` is used to replace calls to this function
+// with a wrapped implementation (defined in gr_graph.h). This ensures that all occurrences of the
+// function in the tested code are redirected to the wrapped versions during unit testing.
+#define rte_node_enqueue_x1 __wrap_rte_node_enqueue_x1
```

Comment:
Can you also move this in `gr_graph.h` and replace all direct includes of `rte_graph.h` or `rte_graph_worker.h` with `gr_graph.h` only so that everyone can benefit from the __wrap_rte_node_enqueue_x1 function?

Reply:
OK, after some discussions with @maxime-leroy we concluded that it is impossible to define this in gr_graph.h otherwise it would replace all occurrences of rte_node_enqueue_x1 in rte_graph.h.